servo/connectors/kubernetes.py

```diff
@@ -1632,6 +1636,15 @@ def pod_template_spec(self) -> kubernetes_asyncio.client.models.V1PodTemplateSpe
         """Return the pod template spec for instances of the Deployment."""
         return self.obj.spec.template
 
+    async def get_tuning_pod_template_spec(self) -> kubernetes_asyncio.client.models.V1PodTemplateSpec:
+        """Return a deep copy of the pod template spec for creation of a tuning pod"""
+        return copy.deepcopy(self.pod_template_spec)
+
+    def update_tuning_pod(self, pod: kubernetes_asyncio.client.models.V1Pod) -> kubernetes_asyncio.client.models.V1Pod:
+        """Update the tuning pod with the latest state of the controller if needed"""
+        # NOTE: Deployment currently needs no updating
+        return pod
+
```

Comment:
How are these tied to tuning? It looks like it doesn't know nor care what Pod it is working against and there's nothing bound to tuning at all